features/ContentCreation.feature

```diff
@@ -0,0 +1,255 @@
+Feature: Content items creation
+  As an administrator
+  In order to menage content to my site
+  I want to create, edit, copy and move content items.
+  
+Background:
+  Given I am logged as "admin"
+    And I go to "Content structure" in "Content" tab
+  
+@javascript @common
+Scenario: Content creation can be cancelled
+  When I start creating a new content "Article"
+	And I set content fields
+      | label | value        |
+      | Title | Test Article |
+	And I click on the edit action bar button "Cancel"
+  Then I should be on content page "eZ Platform" of type "Folder"
+    And there's no "Test Article" "Article" on "eZ Platform" Sub-items list
+
+@javascript @common
+Scenario: Content creation can be closed
+  When I start creating a new content "Article"
+    And I set content fields
+      | label | value        |
+      | Title | Test Article |
+	And I click on the close button
+  Then I should be on content page "eZ Platform" of type "Folder"
+    And there's no "Test Article" "Article" on "Home" Sub-items list
+	
+@javascript @common
+Scenario: Content can be previewed during creation
+  When I start creating a new content "Article"
+	And I set content fields
+		| label | value               |
+		| Title | Test Article2       |
+		| Intro | Test article2 intro |
+	And I click on the edit action bar button "Preview"
+	And I go to "tablet" view in "Test Article2" preview
+	And I go to "mobile" view in "Test Article2" preview
+	And I go to "desktop" view in "Test Article2" preview
+    And I go back from content "Test Article2" preview
+  Then I should be on "Content Update" "Test Article" page
```

Comment:
Actually there was a `assertContains` in this page title verification - I'll change it to `assertStringEndsWith` (cannot be `assertEquals` because of new article case and `New` prefix).